packages/node-cli/src/SdkExecutor.js

```diff
@@ -151,6 +152,17 @@ module.exports = class SdkExecutor {
 		return cliParamsAsString;
 	}
 
+	_getCustomVmOptionsString() {
+		const customVmOptions = this._CLISettingsService.getCustomVmOptions();
+		if (!customVmOptions) {
+			return '';
+		}
+		// customVmOptions are already validated at CLISettingsService 
+		return Object.keys(customVmOptions).reduce((acc, currentKey) =>
```

Comment:
I find the anonymous function hard to read. Extracting it and giving a name could improve code readability

Reply:
Agree. I'll give it a name for better readability .